server/scripts/populate-test-data/pg_dump_test_data.sql

```diff
@@ -31,6 +31,8 @@ $$;
 -- Data for Name: AtVersion; Type: TABLE DATA; Schema: public; Owner: atr
 --
 
+INSERT INTO "AtVersion" ("atId", "name", "releasedAt") VALUES (2, '2023.3.3', '2023-12-02');
+
```

Comment:
Added an AtVersion to the test database so that we can have a supported AtVersion in the DB. This is thought to be safe because it is the test database only but please let me know if this is not the case.

Reply:
Seems fine to me.